sktime/classification/deep_learning/base.py

```diff
@@ -100,6 +100,16 @@ def _predict_proba(self, X, **kwargs):
         probs = probs / probs.sum(axis=1, keepdims=1)
         return probs
 
+    def save(self, path):
+        """Save the model at the given path."""
+        self.model_.save(path)
+
+    def load(self, path):
```

Comment:
should `load` be a method of an estimator? I thought of it as a loose function that produces a fitted estimator.